build/jobs/run-tests.yml

```diff
@@ -32,14 +33,12 @@ jobs:
       azureSubscription: $(ConnectedServiceName)
       KeyVaultName: '${{ parameters.keyVaultName }}-sql'
 
-  - task: VSTest@2
+  - task: DotNetCoreCLI@2
```

Comment:
What is the benefit of this switch? The test run times look about the same and we've lost the test run titles.

Reply:
`VSTest` is windows only. Do you mean the test names in the output?

Reply:
Yes. Previously we had output that looked like this:
![image](https://user-images.githubusercontent.com/17463554/108100367-abf05300-703a-11eb-9e5d-af8e54000af8.png)
But now it looks like this:
![image](https://user-images.githubusercontent.com/17463554/108100449-c6c2c780-703a-11eb-92bc-06c5b566566b.png)

Doesn't effect the outcome, but makes it harder to debug.

Reply:
Thanks for the pickup, looks like I accidentally just removed this line